addons/udes_stock_picking_batch/tests/test_stock_picking_batch.py

```diff
@@ -235,3 +236,418 @@ def test_computing_partial_assignment(self):
         self.complete_pick(self.picking02, call_done=False)
         self.picking02._action_done()
         self.assertEqual(self.batch01.state, "done")
+
+
+class TestBatchMultiDropOff(common.BaseUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestBatchMultiDropOff, cls).setUpClass()
```

Comment:
We can just do `super().setUpClass()` I think.

Reply:
Yes we can do, just copied from udes11 though